website/src/components/EventSummaryCollection/EventSummaryCollection.jsx

```diff
@@ -5,7 +5,7 @@ import './EventSummaryCollection.css';
 
 class EventSummaryCollection extends Component {
   static propTypes = {
-    event: PropTypes.array,
+    event: PropTypes.object,
```

Comment:
PropTypes.object goes against the linter guidlines use `PropTypes.shape({/* Declare the object structure here */})` instead